module2/lessons/advanced_routing.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Advanced Routing
+layout: page
+---
+## Learning Goals
+
+- Understand what the `resources` syntax in `routes.rb` generates for us.
+- Understand what nesting `resources` in `routes.rb` generates for us.
+- Understand the 5 pieces of information `rails routes` gives us.
+- Use route helpers
+- Understand the difference between namespacing and nested `resources``
+
+## Vocabulary
+
+- routes
+- route helper
+- namespace
+
+## Set Up
+For part of this lesson we'll use the [`advanced-routing`](https://github.com/turingschool-examples/set-list-api/tree/advanced-routing-start) branch of the Set List Tutorial.
+
+## Warm Up
+
+In your notebook, without using your computer, fill in the following table for the 5 ReSTful routes for a generic "resource”
+
+Include the following for each:
+
+- Verb
+- URI Pattern
+- Controller#Action
+
+<section class="dropdown">
+### More RESTful Routes  
+
+Did you know...
+
+...when working with Rails as a monolith, using views instead of API endpoints, we actually have two additional RESTful routes.
```

Comment:
This looks great - perfect amount of detail here to just share the background info without diving into these additional CRUD actions